openedx/features/content_tagging/tests/test_rules.py

```diff
@@ -136,7 +140,8 @@ def test_system_taxonomy(self, perm):
         system_taxonomy = api.create_taxonomy(
             name="System Languages",
         )
-        system_taxonomy.system_defined = True
+        system_taxonomy.taxonomy_class = UserSystemDefinedTaxonomy
+        system_taxonomy = system_taxonomy.cast()
```

Comment:
Same suggestion here as with https://github.com/openedx/openedx-learning/pull/67/files#r1271571479

The `rules` themselves should `taxonony.cast()` so we don't have to do this here.. Maybe there's a way to refactor oel_tagging.rules to remove the duplicated code?

Reply:
Don't worry about doing this -- I've done it here: https://github.com/openedx/edx-platform/pull/32661/commits/030bb11c892f25abbf4ce481056654220f6dd19a